lib/node-utils.ts

```diff
@@ -105,4 +105,8 @@ export function hasThenProperty(node: TSESTree.Node) {
 
 export function isArrowFunctionExpression(node: TSESTree.Node): node is TSESTree.ArrowFunctionExpression {
   return node && node.type === 'ArrowFunctionExpression'
+}
+
+export function isObjectExpression(node: TSESTree.Expression): node is TSESTree.ObjectExpression {
+  return node?.type === AST_NODE_TYPES.ObjectExpression
```

Comment:
I found this `enum` which contains all the types we've been hardcoding in this utils function. 

![image](https://user-images.githubusercontent.com/352474/88486752-56e1d800-cf56-11ea-8dfa-7a7e66571419.png)


Perhaps we could do a small technical ticket to refactor these.

Reply:
Yes! I like that idea!

Reply:
Sounds good!

Reply:
#203